src/api/controllers/conversations/conversations.create.js

```diff
@@ -0,0 +1,25 @@
+const errorHandler = require("../../utils/errorHandler")
+const isAuth = require("../../middlewares/isAuth")
+const Sequelize = require("sequelize")
+const log = require("../../utils/log")(module)
+
+module.exports = app => {
+  app.post("/conversations", [isAuth()])
+  app.post("/conversations", async (req, res) => {
+    const { Conversation, User } = req.app.locals.models
+
+    try {
+      const user = await User.findById(req.user.id)
+      const conversations = await Conversation.create({
+        user1 : null,
+        user2 : user.id
```

Comment:
```suggestion
        user2 : req.user.id
```

Voir commentaire au dessus

Reply:
Done.